src/main/java/pl/pjwstk/woloappapi/model/User.java

```diff
@@ -1,8 +1,11 @@
 package pl.pjwstk.woloappapi.model;
 
+import jakarta.persistence.*;
 import lombok.Data;
 
-import jakarta.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
 import java.util.ArrayList;
 import java.util.List;
 
```

Comment:
Zastanawiam się w czy w to jest potrzebne:

```
    @Column(name = "is_pesel_verified", nullable = false)
    private boolean isPeselVerified;
```

(i dla innych pól z typami prymitywnymi - `boolean`. Taka zmienna domyślnie ma wartość `false` nie `null`. Nie da się przypisać do niej wartości `null`. Co innego, gdy zamiast `boolean` [typ prymitywny] chcemy użyć klasy `Boolean`)

https://blog.akai.org.pl/posts/springcourse3/#:~:text=Boolean%20vs%20boolean&text=R%C3%B3%C5%BCnica%20w%20zapisie%20jest%20jednak,obiektem%2C%20a%20boolean%20typem%20prymitywnym.